app/features/conference/components/Conference.js

```diff
@@ -168,6 +168,12 @@ class Conference extends Component<Props, State> {
      * @returns {void}
      */
     _loadConference() {
+        const appProtocolSurplus = `${config.appProtocolPrefix}://`;
+
+        // replace the custom url with https, otherwise new URL() raises 'Invalid URL'.
+        if (this._conference.serverURL.indexOf(appProtocolSurplus) === 0) {
```

Comment:
Meh check with startsWith? Since we only care about it in the start.

Reply:
Done